examples/skywalking-tracing/verify.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+export NAME=skywalking
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+# Waiting SkyWalking server to be ready.
+sleep 60
```

Comment:
use `DELAY` - see https://github.com/envoyproxy/envoy/blob/master/examples/DEVELOPER.md#slow-starting-docker-compositions

that way you will get the `docker ps` *after* the delay and can see the state of the services

Reply:
I will try it. Thanks.